src/main/java/org/sobotics/guttenberg/clients/Guttenberg.java

```diff
@@ -222,6 +222,11 @@ private void update() {
 				}
 				room.leave();
 			}
+			try {
+				wait(10);
+			} catch (InterruptedException e) {
+				e.printStackTrace();
```

Comment:
Typically, this would use a logger as well, otherwise it prints to the console error stream always (and you'd need to override `System.err` to put it in a file for example).